### PR TITLE
copy to clipboard added to onlick chart event

### PR DESCRIPTION
**Pull request by apryka**
## Related tickets?

https://github.com/keen/keen-dataviz.js/issues/191

## How should this be tested?

Step through the code line by line. Things to keep in mind as you review:
 - Are there any edge cases not covered by this code?
 - Does this code follow conventions (naming, formatting, modularization, etc) where applicable?

Fetch the branch and/or deploy to staging to test the following:

- [ ] Does the code compile without warnings (check shell, console)?
- [ ] Do all tests pass?
- [ ] Does the UI, pixel by pixel, look exactly as expected (check various screen sizes, including mobile)?
- [ ] If the feature makes requests from the browser, inspect them in the Web Inspector. Do they look as expected (parameters, headers, etc)?
- [ ] If the feature sends data to Keen, is the data visible in the project if you run an extraction (include link to collection/query)?
- [ ] If the feature saves data to a database, can you confirm the data is indeed created in the database?

### CODE REVIEW

**Comment by adamkasprowicz** [comment]
please add this feature also to the funnels:
if numeric value is present in the step - copy value
if percent - copy percent
if both of them - copy just value

**Comment by adamkasprowicz** [comment]
it would be great if we can sum values if more than one is selected, like here:


sum and copy the result
but don't know if it's even possible with C3 :D

**Comment by adamkasprowicz** [comment]
please add this feature to Metric charts, metric combo,
metric with previous results - you can add 2 click listeners - one on current value, other one on previous value

let's add this also to tables
click on td copies the content of the td

File: lib/index.js [hunk 1]
```diff
@@ -257,15 +261,19 @@ export const Dataviz = function (options = {}) {
   this.config.firstVisibilityState = document.visibilityState;
 
   if (!this.config.utils.clickToCopyToClipboard) {
-    this.config.data.onclick = () => {};
+    this.config.data.onselect = () => {};
```
Comment (adamkasprowicz) [inline lib/index.js]: onselected?

File: lib/libraries/default/funnel-3d.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import * as d3 from 'd3';
 import { textWrap } from '../../utils/svg-text-wrap';
 import { prettyNumber } from '../../utils/pretty-number';
+import copyToClipBoard from '../../utils/copy-to-clipboard';
```
Comment (adamkasprowicz) [inline lib/libraries/default/funnel-3d.js]: no need for Camel that much - clipboard is a single word ;]
copyToClipboard

File: lib/utils/copy-to-clipboard.js [hunk 3]
```diff
@@ -1,8 +1,36 @@
...
   document.execCommand('copy');
   document.body.removeChild(placeholder);
-}
\ No newline at end of file
+
+  if (event) {
+    const notification = document.createElement('div');
+    notification.style.padding = '5px 10px';
+    notification.style.backgroundColor = '#ffffff';
+    notification.style.fontSize = '10px';
```
Comment (adamkasprowicz) [inline lib/utils/copy-to-clipboard.js]: 12px pls

**Comment by apryka** [comment]
feature added to other type of charts.

for c3 charts I decided to add this feature to `onselected` instead of `onclick` event as `click` triggers `select`
